src/sphinxcontrib/towncrier/_towncrier.py

```diff
@@ -0,0 +1,23 @@
+"""Towncrier related shims."""
+
+from pathlib import Path
+from typing import Any, Dict
+
+# pylint: disable=import-error,no-name-in-module
+from towncrier._settings import load_config_from_file  # noqa: WPS436
+
+
+# pylint: disable=fixme
+
+
+def get_towncrier_config(
+        project_path: Path,
+        final_config_path: Path,
+) -> Dict[str, Any]:  # FIXME: add a better type
```

Comment:
I believe the ignore should go here.
```suggestion
) -> Dict[str, Any]:  # FIXME: add a better type  # pylint: disable=fixme
```

Reply:
Sure. I just tried to match the exiting fixme ignore on the other module. Anyway, fixed.

Reply:
That `disable=fixme` in `__init__` on line 85 applies to the comment present on line 86.